### PR TITLE
#229 link uderline and Schedule styleing

### PR DESCRIPTION
**Pull request by baxgas**

### CODE REVIEW

File: app/components/item/Block.vue [hunk 1]
```diff
@@ -44,7 +44,7 @@ export default {
     this.loadNewsImages()
   },
   methods: {
-    async loadNewsImages() {
+    loadNewsImages () {
```
Comment (gammaw) [inline app/components/item/Block.vue:47]: @baxgas why this change?

**Review by gammaw — APPROVED** [review]
Thanks for this too! 

Disclaimer: I didn't try it, only scanned through the changes and commented a question.

File: app/components/item/Block.vue (hunk 1 above)
Comment (baxgas) [reply app/components/item/Block.vue:47]: doesn't need `async`, since it doesn't return a Promise to be resolved and doesn't have `await` for the variables init in the body, so it is not used as an async function. 
don't know why it was put in the first place like this
